pyrex.py

```diff
@@ -213,6 +213,9 @@ def build(args):
                 os.path.join(config['build']['pyrexroot'], 'docker')
                 ]
 
+            if config['pyrex']['registry']:
+                docker_args.extend(['--build-arg', 'MY_REGISTRY=%s/' % config['pyrex']['registry']])
```

Comment:
Okay, one last thing:

Looks to me like the unconditional setter for '--build=arg MY_REGISTRY' (at line 209) needs removed then.

Reply:
Yep, it does. Bad cherry-pick